packages/fxa-auth-server/lib/senders/emails/global.scss

```diff
@@ -84,22 +83,46 @@ tr > td:first-child {
   font-family: $font-sans;
 }
 
-.header-text div {
+.text-header div {
   @extend .text-xl;
   @extend .font-sans;
+  @extend .mb-3;
   text-align: center !important;
-  @extend .mb-5;
 }
 
-.primary-text div {
+.text-body-common {
```

Comment:
If this isn't a class we actually use, we can turn it into a placeholder (`%text-body-common`). 😁